components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -464,6 +467,30 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         return user;
     }
 
+    public async needsVerification(ctx: TraceContext): Promise<boolean> {
+        const user = this.checkUser("needsVerification");
+        return this.verificationService.needsVerification(user);
+    }
+
+    public async sendPhoneNumberVerificationToken(ctx: TraceContext, phoneNumber: string): Promise<void> {
+        this.checkUser("sendPhoneNumberVerificationToken");
```

Comment:
This whole concern feels like it should be part of the EE code-base, and this methods should be empty stubs (like https://github.com/gitpod-io/gitpod/blob/7f2ab4dd96ca9c5494b9fe859601ffbfefe91cfd/components/server/src/workspace/gitpod-server-impl.ts#L2744).

Reply:
What is the reasoning you believe it should be EE? I have no concerns about making this open-source as I don't see how it could be something we want to monetize.

Reply:
Mainly because I thought it would not be that interesting. But have no hard feelings here!

:bulb: Then the binding should move to the other container: https://github.com/gitpod-io/gitpod/pull/11346#discussion_r951072047

Reply:
> My take is to put as much as possible into the open-source version

That has been our strategy so far, yes, and I agree that it applies to phone verification as well. Put it into the "billing" basket at first, but it's unrelated. :100: . Sorry for creating confusion here. :lotus_position: